url-updater/src/main/java/com/devonfw/tools/ide/url/updater/gcloud/GCloudUrlUpdater.java

```diff
@@ -0,0 +1,46 @@
+package com.devonfw.tools.ide.url.updater.gcloud;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.WebsiteUrlUpdater;
+
+import java.util.regex.Pattern;
+
+public class GCloudUrlUpdater extends WebsiteUrlUpdater {
+
+  @Override
+  protected String getTool() {
+
+    return "gcloud";
+  }
+
+  @Override
+  protected void addVersion(UrlVersion urlVersion) {
+
+    String baseUrl = "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${version}-${os}-";
```

Comment:
either declare as local variable or use a constant (`private static final String BASE_URL = ...`).

Reply:
What do you mean by local variable? Is it not already declared locally?